lib/etl/load/create_new_group_records.rb

```diff
@@ -0,0 +1,32 @@
+class CreateNewGroupRecords
+  def write(attributes_list)
+    @attributes_list = attributes_list
+
+    attributes_list.each do |attributes|
+      puts "#{attributes["name"]}"
+      
+      if existing_record attributes
```

Comment:
Maybe we should have parenthesis here, to stay consistent with the rest of the code.